### DISCUSSION TITLE
Window Grouping

### DISCUSSION THREAD

**Discussion by neilroy1998** [lawand-dot-io/taskbar#11] [Feature requests]
Grouping windows of the same app shows them right next to each other, which makes it easy to find the window you want to switch to.

**Comment by lawand-dot-io** [comment]
This has been mentioned by many users, so at this point I plan to add it. Hopefully in one of the upcoming versions :)

**Reply by jameszhan9592** [reply]
Adding my vote for this feature! That said, I would like to be able to ONLY have windows from the same app to be next to each other, without grouping (as in, I don't want windows from the same app to be collapsed into a single app icon, a la Windows' taskbar).

**Reply by lawand-dot-io** [reply]
@jameszhan9592 sure. if I implement collapsing all windows to one icon, I'll add an option in preferences to disable that 👍

**Comment by lawand-dot-io** [comment]
This feature is now almost finished implementing and will be included in the next version. The idea about collapsing the group of windows into a single icon will not yet be implemented, and has been split into a different feature request: https://github.com/lawand-dot-io/taskbar/discussions/50

**Comment by lawand-dot-io** [comment]
This feature has been implemented in Taskbar version 1.2.1 which is now released. Closing the ticket.
